master/buildbot/test/unit/test_status_words.py

```diff
@@ -295,6 +295,13 @@ def test_command_dance(self):
         self.assertTrue(self.sent)  # doesn't matter what it sent
 
     @defer.inlineCallbacks
+    def test_command_hello(self):
+        yield self.do_test_command('hello', exp_usage=False)
+        self.assertEqual(self.sent, ['yes?'])
+        yield self.do_test_command('hello', exp_usage=False)
+        self.assertTrue(self.sent)
```

Comment:
This will be true even if it doesn't send anything new (since it already sent 'yes?').